gripper/firmware/motor_encoder_hardware.c

```diff
@@ -127,7 +123,7 @@ void TIM4_EncoderGSpeed_Init(void) {
     /* Configure TIM4 in Slave mode */
     sSlaveConfig.SlaveMode = TIM_SLAVEMODE_RESET;
     sSlaveConfig.InputTrigger =
-        TIM_TS_ITR1;  // input trigger uses encoder clock
+        TIM_TS_TI1FP1;  // input trigger uses filtered timer input
```

Comment:
What does this do?

Reply:
Not what I thought It did apparently, this makes the crash stop but it is the wrong trigger and breaks encoder_g speed break, gonna re-do and update the PR